indent.go

```diff
@@ -0,0 +1,131 @@
+package main
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"html"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"os/exec"
+	"strings"
+)
+
+type replData struct {
+	id       string
+	language string
+	url      string
+	newUrl   string
+	code     string
+	indented string
+	errors   string
+}
+
+func parseReplItTag(body []byte, tag string) (string, error) {
+	targetS := []byte(fmt.Sprintf("\"%s\":\"", tag))
+	targetE := []byte("\"")
+
+	if start := bytes.Index(body, targetS); start != -1 {
+		if end := bytes.Index(body[start+len(targetS):], targetE); end != -1 {
+			data := string(body[start+len(targetS) : start+len(targetS)+end])
+			return data, nil
+		}
+	}
+	return "", errors.New(fmt.Sprintf("Não foi possível ler a informação '%s'", tag))
```

Comment:
Replace errors.New(fmt.Sprintf...) with fmt.Errorf(...). Also, error messages should always start with lowercase (unless you're doing log.fatal(msg).